notebooks/notebook_template_review.py

```diff
@@ -193,71 +194,81 @@ def parse_notebook(path: str) -> None:
         
         cells = content['cells']
         
-        NotebookRule.init()
+        NotebookRule.init(path, content['cells'])
```

Comment:
You can omit `init`, just do `NotebookRule(...)`

Reply:
1. NotebookRule is an `abstractclass` so should not be instantiated directly (instantiate subclasses of it).
2. Please refactor to pass in cells at the `validate` method. `init` method should have parameters related to the rule definition itself.
3. Pass in cells to the `validate` method.

Reply:
I tried that. Got an error about initializing (__init__) an abstract class. Hence why I went the static method route.

Reply:
done